src/reconstruction/refinement/addReactionGEM.m

```diff
@@ -69,6 +69,8 @@
 end
 
 nMet = length(model.mets)+1;
+orignMets = numel(model.mets);

```

Comment:
Ahh ok, now I get it. yes, you are right, we could simply say "nMet - 1".. updating it.